plugins/inputs/mqtt_consumer/README.md

```diff
@@ -140,5 +140,15 @@ cpu,host=pop-os,tag=telegraf,topic=telegraf/one/cpu/23 value=45,test=23i 1637014
 
 - example when [[inputs.mqtt_consumer.topic_parsing]] is set
 
+- when [[inputs.internal]] is set:
+  - bytes_received (int): count of the number of bytes have been received from incoming messages
+  - messages_received (int): count of the number of messages that have been received from mqtt
+
+## Example Output with [[inputs.internal]] set
+
+```shell
+internal_mqtt_consumer,host=pop-os,version=1.24.0-2a266e53 messages_received=622i,bytes_received=37942i 1657282270000000000
```

Comment:
Add this to the regular example output (and please change the `shell` to `text` as it is not a shell script/command)